### PR TITLE
Add names and types to info structs

### PR DESCRIPTION
**Pull request by bwrsandman**
This is done thanks to the info*.txt files on the mac disk

### CODE REVIEW

File: src/InfoConstants.h [hunk 1]
```diff
@@ -399,23 +405,23 @@ struct CreatureDesireAttributeEntry
...
 {
-	uint32_t field_0x110;
-	uint32_t field_0x114;
-	float field_0x118;
-	uint32_t field_0x11c;
-	float field_0x120;
+	CitadelTypes citadelType;
+	MeshId meshType;
+	float StartLife;
+	uint32_t StartStrength;
```
Comment (bwrsandman) [inline src/InfoConstants.h:423]: lower case

File: src/InfoConstants.h [hunk 2]
```diff
@@ -1292,21 +1292,21 @@ struct GInfluenceInfo
...
+	uint32_t strength;
+	float defence;
+	float startLife;
+	uint32_t growsAfterNumGameTurns;
+	float growthAmount;
+	uint32_t burnCheckAroundEvery;
+	uint32_t randomBurnChance;
+	float minSize;
+	float maxSize;
+	float rainingAcceleratorMultiplier;
```
Comment (raffclar) [inline src/InfoConstants.h:1315]: Spelling mistake

**Review by raffclar — CHANGES_REQUESTED** [review]

File: src/InfoConstants.h (hunk 2 above)
Comment (bwrsandman) [reply src/InfoConstants.h:1315]: what's the mistake?

File: src/AllMeshes.h [hunk 3]
```diff
@@ -1282,4 +1282,455 @@ constexpr std::array<std::string_view, static_cast<uint16_t>(MeshId::_count)> Me
...
+	PWhittlingAtStick = 433,
+	PYankedUnderground = 434,
+	PYankedUnderground2 = 435,
+	PYankedUnderground3 = 436,
+	PYawn = 437,
+	PYawn2 = 438,
+	SPriestSuckLifeForce = 439,
+	AWorm = 440,
+
+	_count
```
Comment (bwrsandman) [inline src/AllMeshes.h:1733]: _COUNT

File: src/Enums.h [hunk 4]
```diff
@@ -111,29 +111,32 @@ enum class VillagerJob
 	Leader = 5,
 	Housewife = 6,
 	Trader = 7,
+
+	_count
```
Comment (bwrsandman) [inline src/Enums.h:115]: _COUNT

File: src/Enums.h [hunk 5]
```diff
@@ -1378,14 +1641,14 @@ enum class PlaytimeInfo
...
 {
 	None = -1,
 	Normal = 0,
 	Abode = 1,
 };
 
-enum class CountryList
+enum class CountryList : uint32_t
 {
 	None = 0,
```
Comment (bwrsandman) [inline src/Enums.h:1653]: Aren't there names for these countries?

File: src/Enums.h [hunk 6]
```diff
@@ -2672,4 +2935,184 @@ static constexpr std::array<std::string_view, static_cast<uint8_t>(PlayerNames::
...
+	Walkway = 5,
+};
+
+enum class AttackInfo : int
+{
+	None = -1,
+	HandToHand = 0,
+	Arrow = 1,
+
+	_Last = 2
```
Comment (bwrsandman) [inline src/Enums.h:3099]: _COUNT

File: src/Enums.h [hunk 7]
```diff
@@ -2672,4 +2935,184 @@ static constexpr std::array<std::string_view, static_cast<uint8_t>(PlayerNames::
...
+	Spell = 2,
+	Animal = 3,
+	Chant = 4,
+	PlayerInteraction = 5,
+	PlayerInteractionDrown = 6,
+	Sacrifice = 7,
+	Exhaustion = 8,
+	OldAge = 9,
+	Last = 10,
+	_COUNT
```
Comment (bwrsandman) [inline src/Enums.h:3115]: * Death reason already exists
* is count the same as last?

File: src/Enums.h (hunk 5 above)
Comment (bwrsandman) [reply src/Enums.h:1657]: Nope

**Pull request merged**